addons/pos_restaurant/static/src/js/Chrome.js

```diff
@@ -65,7 +65,7 @@ odoo.define('pos_restaurant.chrome', function (require) {
                 }
             }
             _actionAfterIdle() {
-                if (this.tempScreen.isShown) {
+                if (this.tempScreen.isShown && this.tempScreen.name !== 'LoginScreen') {
```

Comment:
`LoginScreen` is defined in `pos_hr` module.
It's probably cleaner to override `_actionAfterIdle` in `pos_hr` module.